src/support/react-docs/react-docs.js

```diff
@@ -0,0 +1,163 @@
+const path = require('path')
+const { reporter } = require('@dhis2/cli-helpers-engine')
+const fs = require('fs-extra')
+const glob = require('glob')
+const reactDocgen = require('react-docgen')
+const urlJoin = require('url-join')
+const findExportedComponentDefinitionsUsingImporter = require('./custom-resolver')
+const filepathHandler = require('./filepath-handler')
+const getHTMLPropTable = require('./prop-table')
+
+/*
+ * Parses a React Docgen docs object and makes nice-looking markdown.
+ * @param {Object} docgenDocs - a docs object that is the result of React Docgen parsing a file
+ * @param {Object} options - { linkSource: boolean, remoteRepoURL: string, localRepoRoot: string }
+ * @returns {string} Markdown describing the component that produced the docs object
+ */
+function getMarkdownFromDocgen(docgenDocs, options) {
+    const { linkSource, remoteRepoURL, localRepoRoot } = options
+    let sourceURL
+    if (linkSource) {
+        // handles case where script is run in a different dir than repo root
+        const relativeFilepath = path.relative(
+            localRepoRoot,
+            docgenDocs.filepath
+        )
+        sourceURL = urlJoin(remoteRepoURL, relativeFilepath)
+    }
+
+    const displayName = `### ${docgenDocs.displayName}`
+    // filepath [docgenDocs.filepath] - added by filepath handler
+    const filepath = linkSource
+        ? `[**${docgenDocs.filepath}**](${sourceURL})`
+        : `**${docgenDocs.filepath}**`
```

Comment:
I'm not sure if this due to updating `react-docgen` to `v6.0.0` or some other change, but since #116 the component filepaths shown below the headings are absolute paths. The following fix undoes that change:

```diff
diff --git a/src/support/react-docs/react-docs.js b/src/support/react-docs/react-docs.js
index f097a32..9e1bf08 100644
--- a/src/support/react-docs/react-docs.js
+++ b/src/support/react-docs/react-docs.js
@@ -17,9 +17,10 @@ const getHTMLPropTable = require('./prop-table')
 function getMarkdownFromDocgen(docgenDocs, options) {
     const { linkSource, remoteRepoURL, localRepoRoot } = options
     let sourceURL
+    let relativeFilepath
     if (linkSource) {
         // handles case where script is run in a different dir than repo root
-        const relativeFilepath = path.relative(
+        relativeFilepath = path.relative(
             localRepoRoot,
             docgenDocs.filepath
         )
@@ -29,7 +30,7 @@ function getMarkdownFromDocgen(docgenDocs, options) {
     const displayName = `### ${docgenDocs.displayName}`
     // filepath [docgenDocs.filepath] - added by filepath handler
     const filepath = linkSource
-        ? `[**${docgenDocs.filepath}**](${sourceURL})`
+        ? `[**${relativeFilepath}**](${sourceURL})`
         : `**${docgenDocs.filepath}**`
     const composes = docgenDocs.composes
         ? `Composes ${docgenDocs.composes.join(', ')}`
```

Reply:
Ah yeah, good catch - the filepaths are handled differently now that components are found by traversing imports instead of walking files